apputils/config_manager.py

```diff
@@ -0,0 +1,83 @@
+"""Configuration Manager."""
+from celery.schedules import crontab
+from config_validator import ConfigValidator
+from string import Template
+import ConfigParser
+import os
+
+
+class ConfigManager(object):
+    def __init__(self, config_path):
+        self.scheduled_tasks = {}
+        self.load_config_files(config_path)
+
+    def load_config_files(self, config_path):
+        """Load all valid config files from path, print task config to stdout.
+
+        Args:
+            config_path(str): Path to configuration directory.
+
+        """
+        files_in_dir = [os.path.join(config_path, f)
+                        for f in os.listdir(config_path)
+                        if os.path.isfile(os.path.join(config_path, f))]
+        for target in sorted(files_in_dir):
```

Comment:
why are we sorting `files_in_dir`

Reply:
We sort the files so that they're loaded in a predictable order.  Makes behavior more deterministic.

Reply:
why not use `ordereddict`?

Reply:
Because it's not a dictionary, it's a list.  There's no need for key-value pairs in this part of the code. Using `ordereddict` would still require sorting the contents, if you want the contents in alpha order.
https://docs.python.org/2/library/collections.html#collections.OrderedDict